src/ampl.js

```diff
@@ -1,33 +1,29 @@
-var Remarkable = require('remarkable');
-var htmlparser = require("htmlparser2");
```

Comment:
*Out with the old, I say*!
https://twitter.com/ericdfields/status/677677470590570496/photo/1